controllers/books.js

```diff
@@ -0,0 +1,122 @@
+const { Book } = require('../models')
+
+class bookController {
+    static getAllBook (req, res) {
+        let where = {}
+        if(req.query) {
+            let key = Object.keys(req.query)
+            let temp= []
+            key.forEach(e => {
+                temp.push({ [e]: { $regex: new RegExp(req.query[e], 'i') } })
+            })
+            if(temp.length > 0) {
+                where = {
+                    $or: temp
+                }
+            }
+        }
+        Book.find(where)
+        .then(books => {
```

Comment:
Indentasi nya dirapihkan lagi

```suggestion
           .then(books => {
```